src/main.ts

```diff
@@ -13,15 +14,27 @@ config.load();
 const container = new Container();
 
 let store: PersistentStorage<number|string>;
+let DBStore: any;
+
+switch(config.dbMode) {
+    case ConfigManager.PG:
+        DBStore = PostgresStore;
```

Comment:
use IoC container to bind PostgresStore to PersistentStorage instead of replace variable.